queue_services/entity-emailer/.env.sample

```diff
@@ -0,0 +1,84 @@
+
+# Flask
+FLASK_ENV=
+
+ACCOUNT_SVC_AUTH_URL=
+ACCOUNT_SVC_CLIENT_ID=
+ACCOUNT_SVC_CLIENT_SECRET=
+APP_FILE=
+AUTH_URL=
+DASHBOARD_URL=
+
+
+
+LEGAL_API_URL=
+#LEGAL_API_URL=
+NOTIFY_API_URL=
+#NOTIFY_API_URL=
+PAY_API_URL=
+#PAY_API_URL=
+#SENTRY_DSN=
+TEMPLATE_PATH=
+
+
+NAMEX_AUTH_SVC_URL=
+NAMEX_SERVICE_CLIENT_SECRET=
+NAMEX_SERVICE_CLIENT_USERNAME=
+NAMEX_SVC_URL=
+
+# namex config TEST
+#NAMEX_AUTH_SVC_URL=
+#NAMEX_SERVICE_CLIENT_USERNAME=
+#NAMEX_SERVICE_CLIENT_SECRET=
+#NAMEX_SVC_URL=
+
+
+### SQL Alchemy
+DATABASE_USERNAME=
+DATABASE_PASSWORD=
+DATABASE_NAME=
+DATABASE_HOST=
+DATABASE_PORT=
+
+
+DATABASE_TEST_USERNAME=
+DATABASE_TEST_PASSWORD=
+DATABASE_TEST_NAME=
+DATABASE_TEST_HOST=
+DATABASE_TEST_PORT=
+
+
+TRACKER_DATABASE_USERNAME=
+TRACKER_DATABASE_PASSWORD=
+TRACKER_DATABASE_NAME=
+TRACKER_DATABASE_HOST=
+TRACKER_DATABASE_PORT=
+
+
+TRACKER_DATABASE_TEST_USERNAME=
+TRACKER_DATABASE_TEST_PASSWORD=
+TRACKER_DATABASE_TEST_NAME=
+TRACKER_DATABASE_TEST_HOST=
+TRACKER_DATABASE_TEST_PORT=
+
+
+
+## ## NATS - STAN
+NATS_SERVERS=
+NATS_CLIENT_NAME=
+NATS_CLUSTER_ID=
+NATS_EMAILER_SUBJECT=
+NATS_ENTITY_EVENT_SUBJECT=
+#NATS_QUEUE=
+NATS_QUEUE=
+STAN_CLUSTER_NAME=
+
+# ## NATS - STAN - DEV
+#NATS_SERVERS=
+#NATS_CLIENT_NAME=
+#NATS_CLUSTER_ID=
+#NATS_EMAILER_SUBJECT=
+#NATS_ENTITY_EVENT_SUBJECT=
+#NATS_QUEUE=
+
+#DEPLOYMENT_ENV=
```

Comment:
If an .env file is needed, then I suggest we include a sanitized copy so it's more obvious that an .env is required.